include/ddog_profiling_utils.hpp

```diff
@@ -12,6 +12,11 @@ extern "C" {
 #include "string_view.hpp"
 
 #include <string_view>
+#include <span>
+
+inline ddog_CharSlice to_CharSlice(std::span <const unsigned char>span) {
```

Comment:
This function seems useless (buildid is a `std::string` in MapInfo, and therefore `ddog_CharSlice to_CharSlice(std::string_view str)` should be able to convert it to a slice).